requirements/constraints.txt

```diff
@@ -30,11 +30,11 @@ django-storages==1.9.1
 # for them.
 edx-enterprise==4.0.16
 
-# oauthlib>3.0.1 causes test failures ( also remove the django-oauth-toolkit constraint when this is fixed )
-oauthlib==3.0.1
-
-# django-auth-toolkit==1.3.3 requires oauthlib>=3.1.0 which is pinned because of test failures
-django-oauth-toolkit<=1.3.2
+# django-oauth-toolkit version >2.0.0 has breaking changes.
+# More details mentioned on this issue https://github.com/openedx/edx-platform/issues/32884
+# Versions from 1.3.3 to 2.0.0 have some migrations related changes.
+# so we're upgrading minor versions one by one.
+django-oauth-toolkit==1.4.1
```

Comment:
Proposed update to:
a. Make it more clear that these are two different points (although related), and
b. Replace 1.X.X with the next version that has migrations that you wish to protect against. Note that we'll already be past 1.3.3 in this PR, so it is already irrelevant.
```suggestion
# 1. django-oauth-toolkit version >2.0.0 has breaking changes. More details
#    mentioned on this issue https://github.com/openedx/edx-platform/issues/32884
# 2. Versions from 1.X.X to 2.0.0 have some migrations related changes.
#    so we're upgrading minor versions one by one.
django-oauth-toolkit<1.X.X
```